src/core/lv_obj_event.c

```diff
@@ -378,6 +380,23 @@ static lv_result_t event_send_core(lv_event_t * e)
         res = event_send_core(e);
     }
 
+    /*Trickle down to children if enabled*/
+    if(event_is_trickled(e)) {
+        uint32_t child_count = lv_obj_get_child_count(target);
+
+        /* we don't want the event to bubble up again when trickling down */
+        e->stop_bubbling = 1;
+
+        for(uint32_t i = 0; i < child_count && res == LV_RESULT_OK && !e->stop_processing; i++) {
+            lv_obj_t * child = lv_obj_get_child(target, i);
+            if(child) {
+                e->current_target = child;
+                res = event_send_core(e);
+                if(res != LV_RESULT_OK) break;
```

Comment:
I would suggest to put an else with a LOG_WARN / ERR , it would be good to catch cases when sending event to child objects fail.